sources/org.osbuild.curl

```diff
@@ -217,7 +218,7 @@ class CurlSource(sources.SourceService):
                     "--location",
                     "--config", curl_config_path,
                 ]
-                if _curl_has_parallel_downloads():
+                if curl_has_parallel_downloads():
```

Comment:
This should also be checking the cached value, which will cut forks in half :)

Reply:
This is also reworked in the latest iteration :)